games-roguelike/hengband/files/hengband-1.6.2-autoconf-ncurses.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/configure.in b/configure.in
+index 0df0e76..806279a 100644
+--- a/configure.in
++++ b/configure.in
+@@ -56,7 +56,7 @@ fi
+ 
+ dnl Checks for libraries.
+ dnl Replace `main' with a function in -lncurses:
+-AC_CHECK_LIB(ncurses, initscr, [AC_DEFINE(USE_GCU, 1, [Allow -mGCU environment]) AC_DEFINE(USE_NCURSES, 1, [Use ncurses]) LIBS="$LIBS -lncurses"])
++AC_CHECK_LIB(ncurses, initscr, [AC_DEFINE(USE_GCU, 1, [Allow -mGCU environment]) AC_DEFINE(USE_NCURSES, 1, [Use ncurses]) LIBS="$LIBS -lncurses -ltinfo"])
```

Comment:
I think we might technically need nursesw and so on (wide) for USE=unicode, which is now forced on. But in order to avoid forcing USE=tinfo (which is also forced on, but still), we should use PKG_CHECK_MODULES instead to determine the LIBS content based on what the system needs.

Reply:
Here's an example: 4720158264396a3ffbe291018c5a8d81fa2fc31b.

Reply:
Thanks for the example, I have fixed this to use `PKG_CHECK_MODULES`.